util/mbed/mbed_cmake_vargen.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
```

Comment:
With the "shebang" set like this, it will run on some systems with python2 and others with python3.  This is because on many newer systems, `python` is just a symlink to `python3`.

This script looks like it only works with python2, so if you cange the first line to `#!/usr/bin/env python2`, it will work on all systems without issue.